ExtraEnemyCustomization/EntryPoint.cs

```diff
@@ -35,6 +35,8 @@ public override void Load()
 
             Logger.LogInstance = Log;
 
+            var useLiveEdit = Config.Bind(new ConfigDefinition("General", "Live Edit"), false, new ConfigDescription("Reload Config when they are edited while in-game"));
+            var linkMTFOHotReload = Config.Bind(new ConfigDefinition("General", "Reload on MTFO HotReload"), false, new ConfigDescription("Reload Configs when MTFO's HotReload button has pressed?"));
```

Comment:
Make it active by default
```suggestion
            var linkMTFOHotReload = Config.Bind(new ConfigDefinition("General", "Reload on MTFO HotReload"), true, new ConfigDescription("Reload Configs when MTFO's HotReload button has pressed?"));
```